rfc-035-use-standard-linting-tools-across-all-our-projects.md

```diff
@@ -0,0 +1,37 @@
+# Use standard linting tools across all of our projects
+
+## Summary
+
+All dxw-owned respositories MUST use a standard linting tool for the
+main programming language the repo is written in, where one is available.
+
+## Problem
+
+As developers, we commonly have different styles of writing code. Unchecked
+this can lead to inconsistencies in style, which over time can make code
+messy, and difficult to maintain and review. It can also lead to bikeshedding
+over code style in code reviews, which can cause conflict and slow down code
+review.
+
+## Proposal
+
+Where available, all new dxw-owned or maintained repositories MUST use a
+standard linting tool to make sure code meets a standard style. Examples
```

Comment:
What is a "_standard_ linting tool"? If we just mean a linting tool, I recommend removing the "standard". If we don't, I suggest using clearer language.

Reply:
I think there's some value in us having dxw standards. So StandardRB for Ruby, ESLint and Prettier with one of a specific set of presets for JavaScript, Prettier for everything it will format, whatever we use for PHP, etc.

Reply:
Ah, if that's the meaning then I'd suggest changing the wording to be more along the lines of "repositories SHOULD use a linting tool, and that linting tool SHOULD be one on the below list".

At the moment the text says the below list are examples of linting tools, not linting tools that SHOULD be used.

Reply:
I suggest that that list lives somewhere else, too, or we're going to have it fractured across many RFCs in the long run.